resources/public/api/conf/1.0/lossClaimsOrder_amend.raml

```diff
@@ -0,0 +1,30 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.formatClaimType
+  - errors.formatSequence
+  - errors.RuleInvalidSequenceStart
+  - errors.RuleSequenceOrderBroken
+  - errors.MissingMandatoryField
+  - errors.RuleLossClaimsMissing
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFoundClaimId
+  - queryParameters.taxYearForClaims
+displayName: Amend Loss Claims Order
+description: This endpoint allows the developer to change the sequence in which certain losses are used.
+(annotations.sandboxData): !include scenarios/lossClaimsOrder_amend.md
+(annotations.scope): "write:self-assessment"
+securedBy: [ sec.oauth_2_0: { scopes: [ "write:self-assessment" ] } ]
```

Comment:
I believe within the spec it says the scope is "read:self-assessment" so the write should be changed here.